### DISCUSSION TITLE
Clearing template cache when saving a form

### DISCUSSION THREAD

**Discussion by alexjcollins** [solspace/craft-freeform#197] [Questions & Answers] [answered]
Hey,

I saw a [GitHub issue](https://github.com/solspace/craft3-freeform/issues/35) stating that since forms aren't elements, this wasn't possible - but I wonder if you have any advice on what we can do instead?

We have a client that's using a form across a number of different pages. At the moment, when making adjustments to the form they're having to either dump the entire template cache or go into each of the pages using the form and save them so that the template cash is refreshed.

We're using Blitz, by the way.

Cheers!

**Comment by kjmartens** [comment, marked as answer]
Unfortunately I'm not really sure about how this could be automated in any way. I did notice this [Buster plugin](https://plugins.craftcms.com/buster) in the Plugin Store that appears to let you manually clear the cache of specific pages, etc. No idea if it works at all with Blitz.

**Reply by alexjcollins** [reply]
Thanks for the reply - we'll give it a shot.
If you do manage to figure out a first-party solution it would be massively appreciated.
